### DISCUSSION TITLE
[QUESTION] | Measuring Full Page Load time

### DISCUSSION THREAD

**Discussion by hayk96** [GoogleChrome/lighthouse#12939] [Q&A] [answered]
Hello dear community👋   
You made a nice tool and I'm happy using it in a work. I have a question related to the Full Page Load. I want to know which timing entries of the components reported by Lighthouse I have to use for calculating Full page Load time.

**Comment by brendankenny** [comment, marked as answer]
Hi @Hayk96, 

>  I want to know which timing entries of the components reported by Lighthouse I have to use for calculating Full page Load time.

It depends on what you mean by "Full Page Load". Is there a specific definition you had in mind?

You could use one of two of the top-level metrics in the Lighthouse performance section:
- Largest Contentful Paint (`lhrJson.audits['largest-contentful-paint'].numericValue` in the JSON lighthouse result) measures when the site shows it's biggest asset (usually a big image or headline text), which correlates well with when users _perceive_ the page to have finished loading
- Time to Interactive (`lhrJson.audits.interactive.numericValue` in the JSON lighthouse result) tries to measure when network and CPU activity have quieted down and so indicate the browser is probably finished doing work to load the page

but as I said, there are multiple things you could mean here, but I'd need more information to know what you mean.

**Reply by hayk96** [reply]
Hi @brendankenny,
Thanks for the reply. I'm sorry for my late response. I've used the `speed-index` as an indicator for measuring the "Full Page Load".
